### DISCUSSION TITLE
Can't resolve images in CSS (background image, etc)

### DISCUSSION THREAD

**Discussion by tr1s** [vercel/next.js#22185] [Help] [answered]
<img width="1003" alt="CleanShot 2021-02-15 at 01 11 32@2x" src="https://user-images.githubusercontent.com/19172250/107911568-dce16280-6f2a-11eb-9e08-bb418eb055a4.png">
<img width="713" alt="CleanShot 2021-02-15 at 01 12 30@2x" src="https://user-images.githubusercontent.com/19172250/107911589-e5d23400-6f2a-11eb-92c5-3ef104a387ea.png">

I thought this would be so simple but I can't figure it out. What do I have to do to make this work?

**Comment by tr1s** [comment, marked as answer]
I figured it out by moving these files out of the public folder. It seems I have a fundamental misunderstanding, could someone help explain this to me?

I put all my images in public because of `next/image`, and that lets me using them in `<Image src="/path-to-file" />` .. But this means they are not accessible in CSS the same way?

Would love some clarification!

I'm also noticing this works:

`content: url('../../images/quote-white.svg');`

but not this:

`content: url('/src/images/quote-white.svg');`

**Reply by fkapsahili** [reply]
The following source should work with your example above:

`url(„/images/night.jpg“)`

**Reply by tr1s** [reply]
Yes it does, I mentioned that. It's using the `public` folder and/or starting with a `/` that doesn't work. Just curious as to why and looking for understanding around it all.

**Reply by renerubio** [reply]
@tr1s @fkapsahili 
is there any documentation on this behavior? 
it is curious that using the public folder and/or starting with a / this does not work with CSS.

**Comment by tr1s** [comment]
All good here now!
